akka-persistence/src/main/scala/akka/persistence/Persistence.scala

```diff
@@ -105,8 +105,18 @@ trait PersistenceRecovery {
    * To skip recovery completely return `Recovery.none`.
    */
   def recovery: Recovery = Recovery()
-  //#persistence-recovery
 }
+//#persistence-recovery
+
+//#persistence-stash
+trait PersistenceStash extends Stash with StashFactory {
+  /**
+   * The returned [[StashOverflowStrategy]] object determine how to handle the received message when the
+   * size of Stash exceeds the capacity of the Stash.
+   */
+  def stashOverflowStrategy: StashOverflowStrategy = ThrowOverflowExceptionStrategy
```

Comment:
we should mention this method in the reference documentation, there is already some words about the stash in the docs, perhaps we should move that to a new section and include this